src/client/static/analytics/ophan.ts

```diff
@@ -1,4 +1,26 @@
 import { record } from '@/client/lib/ophan';
+import { storage } from '@guardian/libs';
+
+const addRefToOphanFollow = () => {
+  if (storage.local.isAvailable()) {
+    const { search } = window.location;
+    const params = new URLSearchParams(search);
+    const ref = params.get('ref');
+    const refViewId = params.get('refViewId');
+
+    if (ref && refViewId) {
+      storage.local.setRaw(
+        'ophan_follow',
```

Comment:
one small q, could we be overwriting a value already set against `ophan_follow` in local storage here?

Reply:
I dont think so, as it seems it's hardly used: https://github.com/search?q=org%3Aguardian+ophan_follow&type=code